cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -86,7 +86,7 @@ resource volumesAccountRoleAssignment 'Microsoft.Authorization/roleAssignments@2
 {{- range $volume := $value.Volumes}}
 resource {{mergeBicepName $name $volume.Name}}FileShare 'Microsoft.Storage/storageAccounts/fileServices/shares@2022-05-01' = {
   parent: storageVolumeFileService
-  name: take('${toLower('{{$name}}')}-${toLower('{{removeDot $volume.Name}}')}', 32)
+  name: take('${toLower('{{$name}}')}-${toLower('{{removeDot $volume.Name}}')}', 60)
```

Comment:
I'm not sure if we are worried about this or not but the physical resource name will change because of this when someone updates their copy of `azd` (if they had not run `infra synth`).  Are we okay with that?

Reply:
I think it is fine.  Volumes are not very popular yet (b/c file system issues) and bindMounts are alpha.

Reply:
We made some changes to aspire 9 that result in simmilar problems. We gave people a way to opt back into v8 naming schemes though. Not sure if it's worth it for volumes.